proto/cosmos/autocli/v1/options.proto

```diff
@@ -101,17 +101,14 @@ message FlagOptions {
   // default_value is the default value as text.
   string default_value = 4;
 
-  // default value is the default value as text if the flag is used without any value.
-  string no_opt_default_value = 5;
-
   // deprecated is the usage text to show if this flag is deprecated.
-  string deprecated = 6;
+  string deprecated = 5;
```

Comment:
Decreased these too, as per https://github.com/cosmos/cosmos-sdk/pull/14985#discussion_r1102044532.
Main needs an extra PR to do the same.